runtime/development/src/lib.rs

```diff
@@ -835,23 +888,29 @@ parameter_types! {
 	pub const MaxSubAccounts: u32 = 100;
 	pub const MaxAdditionalFields: u32 = 100;
 	pub const BasicDeposit: Balance = 100 * CFG;
-	pub const FieldDeposit: Balance = 25 * CFG;
+	pub const ByteDeposit: Balance = deposit(0, 1); //TODO-1.7
```

Comment:
TODO: Check if we need to change this value

Reply:
It seems the deposit is now computed by byte instead of by the entire field, so this constant must be less than before